helm/scylla-operator/templates/webhookserver.deployment.yaml

```diff
@@ -37,6 +37,7 @@ spec:
         - --loglevel=2
         - --tls-cert-file=/tmp/k8s-webhook-server/serving-certs/tls.crt
         - --tls-private-key-file=/tmp/k8s-webhook-server/serving-certs/tls.key
+        - --port=5000
```

Comment:
question: what's the purpose of the `SCYLLA_OPERATOR_PORT` env var? Why is it done on the application level and not in a k8s' service?